docs/guide/rules.md

```diff
@@ -379,6 +379,13 @@ The field under validation must be an iterable and/or have a length property of
 <input v-validate="'length:5'" type="text" name="field">
 ```
 
+Specifying both len and max:
+
+```html
+<input v-validate="'length:[2, 3]'" type="checkbox" name="field">
```

Comment:
Shouldn't this be `v-validate="'length:2,3'"` there shouldn't be square brackets.

Reply:
I noticed you only need the square brackets when using the object-form. `{length: [2, 3]}`. Thanks. The _string form_ should be enough for the doc now, but in the future, we will want to show both forms of construct. It gets confusing sometimes.